lib/resolvers/pil.js

```diff
@@ -3,7 +3,26 @@ const { pick } = require('lodash');
 const { generateLicenceNumber } = require('../utils');
 
 module.exports = ({ models }) => ({ action, data, id }, transaction) => {
-  const { PIL } = models;
+  const { PIL, PilTransfer } = models;
+
+  if (action === 'transfer') {
+    return PIL.query(transaction).findById(id)
+      .then(pil => pil.$query(transaction).patchAndFetch({
```

Comment:
What's the difference between `.query()` and `.$query()`?

Reply:
`.query()` is on the Model class, `.$query()` is on an instance of the Model so it knows which record to update.